images/kubekins-e2e/Makefile

```diff
@@ -17,23 +17,23 @@ TAG := $(shell date +v%Y%m%d)-$(shell git describe --tags --always --dirty)
 
 # kubernetes master branch config
 K8S ?= master
-GO ?= 1.10.2
+GO ?= 1.10.4
 BAZEL ?= 0.16.1
 UPGRADE_DOCKER ?=false
 # TODO(bentheelder,cblecker): this is aging and bad
 CFSSL ?= R1.2
 
 # config for testing prior to rolling out to master / ongoing release
 ifeq ($(K8S), experimental)
-	GO = 1.10.2
+	GO = 1.10.4
```

Comment:
as mentioned, we should upgrade this to the latest stable go, but i'll do this in another PR, experimental images are orthogonal here.